symengine/tests/basic/test_functions.cpp

```diff
@@ -1123,6 +1123,64 @@ TEST_CASE("Sin table: functions", "[functions]")
 
 }
 
+TEST_CASE("Cos table: functions", "[functions]")
+{
+    RCP<const Basic> r1;
+    RCP<const Basic> r2;
+
+    RCP<const Basic> i2 = integer(2);
+    RCP<const Basic> i3 = integer(3);
+    RCP<const Basic> i12 = integer(12);
+    RCP<const Basic> i13 = integer(13);
+    RCP<const Basic> im1 = integer(-1);
+
+    RCP<const Symbol> x = symbol("x");
+    RCP<const Symbol> y = symbol("y");
```

Comment:
Looks like `x` and `y` are not used in the tests. You can remove these lines if not necessary.